.github/testoolScripts/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -x
+
+export PATH=/home/ubuntu/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/usr/local/go/bin/
```

Comment:
Do we have games?

```suggestion
export PATH=/home/ubuntu/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/snap/bin:/usr/local/go/bin/
```

Reply:
Just the folder is there, but i agree there is no need to have in the path!